.github/pull_request_template.md

```diff
@@ -1,3 +1,13 @@
+<!--
+
+If you're opening a PR to cherry-pick a change for a release branch, use this template instead:
+https://github.com/apple/swift-docc/blob/main/.github/PULL_REQUEST_TEMPLATE/CHERRY_PICK.md
+
+Curretnly, Github website does not have a UX for it.
+You can manually set the template on qurey parmas eg. `https://github.com/apple/swift-docc/compare/release/5.7...apple:main?template=CHERRY_PICK.md`
+or use GitHub CLI - [gh](https://cli.github.com) to choose all the PR templates.
```

Comment:
IMO I don't think the second paragraph here is needed, I'd rather keep it simple with the first paragraph. If people want to look up PR template documentation they can find it themselves.

Reply:
Updated.